### PR TITLE
update(games): field name and is_guest_ai ,add check user has already…

### PR DESCRIPTION
**Pull request by C-Kang17**
… created a game, they won't create a game

### CODE REVIEW

File: app/modules/v1/games/exceptions.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from core.exceptions import ErrorCode as CoreErrorCode
+from exceptions import CustomException
+
+
+class ErrorCode(CoreErrorCode):
+    @staticmethod
+    def InvalidCreateGame():
+        return CustomException(type="games/info/invalid-create-games", status=400, title="Invalid create game.", detail="The game has already been created.")
```
Comment (ellyx13) [inline app/modules/v1/games/exceptions.py]: games/warning/already-game

File: app/modules/v1/games/exceptions.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+from core.exceptions import ErrorCode as CoreErrorCode
+from exceptions import CustomException
+
+
+class ErrorCode(CoreErrorCode):
+    @staticmethod
+    def InvalidCreateGame():
```
Comment (ellyx13) [inline app/modules/v1/games/exceptions.py]: AlreadyGame

File: app/modules/v1/games/services.py [hunk 3]
```diff
@@ -18,6 +18,12 @@ async def create(self, data: schemas.CreateRequest, commons: CommonsDependencies
         data["created_by"] = data["host_id"] = self.get_current_user(commons=commons)
         data["created_at"] = self.get_current_datetime()
         data_save = models.Games(**data).model_dump()
+        # Check if the user has already created a game, they will not be allowed to create another one.
+        data_check = await self.crud.get_by_field(data=data_save["host_id"], field_name="host_id")
+        if data_check:
+            for record in data_check:
+                if record["status"] in ["waiting" ,"in_progress"]:
+                    raise exceptions.ErrorCode.InvalidCreateGame()
```
Comment (ellyx13) [inline app/modules/v1/games/services.py]: Dùng exception sai cách rồi, nhìn bên users dùng sao r dùng lại như v. Import exception as abc

Comment (C-Kang17) [reply app/modules/v1/games/services.py]: oke

**Pull request merged**